src/en/weebcentral/src/eu/kanade/tachiyomi/extension/en/weebcentral/WeebCentral.kt

```diff
@@ -102,13 +102,24 @@ class WeebCentral : ParsedHttpSource() {
         with(document.select("section[x-data] > section")[0]) {
             thumbnail_url = selectFirst("img")!!.attr("abs:src")
             author = select("ul > li:has(strong:contains(Author)) > span > a").joinToString { it.text() }
-            genre = select("ul > li:has(strong:contains(Tag)) > span > a").joinToString { it.text() }
+            genre = selectFirst("ul > li:has(strong:contains(Type)) > a")!!.text()
+            genre += ", " + select("ul > li:has(strong:contains(Tag)) > span > a").joinToString { it.text() }
             status = selectFirst("ul > li:has(strong:contains(Status)) > a").parseStatus()
+            description = ""
+            if (selectFirst("""ul > li:has(a[href$="official=True"])""") != null) {
```

Comment:
Use `~=` (contains) if there is no better selector. Usually you would use `'` for CSS to avoid string escaping issues.